ucsdscheduleme/Repo/FormatRepo.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ucsdscheduleme.Models;
+
+namespace ucsdscheduleme.Repo
+{
+    public class FormatRepo
+    {
+        private static readonly int START_HOUR = 7;
+        private static readonly int START_MINUTES = 30;
+        private static readonly int MINUTES_IN_HOUR = 60;
+
+        public static ScheduleViewModel FormatSectionsToCalendarEvent(List<Section> sections)
+        {
+
+            ScheduleViewModel model = new ScheduleViewModel();
+            List<CalendarEvent> events = new List<CalendarEvent>();
+            List<OneTimeEvent> oneTimeEvents = new List<OneTimeEvent>();
+            List<Metadata> classMetadata = new List<Metadata>();
+            Metadata overall = new Metadata();
+
```

Comment:
Add comments to the effect of "initializing locals" and "defining objects" or something like that, just for ease of skimming.